dpnp/backend/CMakeLists.txt

```diff
@@ -288,3 +288,7 @@ message(STATUS "CMAKE_LINKER:                    ${CMAKE_LINKER}")
 message(STATUS "CMAKE_SOURCE_DIR:                ${CMAKE_SOURCE_DIR}")
 message(STATUS "DPNP_INSTALL_PREFIX:             ${CMAKE_INSTALL_PREFIX}")
 message(STATUS "CMAKE_VERBOSE_MAKEFILE:          ${CMAKE_VERBOSE_MAKEFILE}")
+
+if(ENABLE_TESTS)
```

Comment:
```suggestion
if(DPNP_BACKEND_TESTS)
```

Reply:
Also, make it as `DPNP_DEBUG_ENABLE ` and move this code to the top of the file near line 50